integr8ly/installation
# INTLY-1260 Remove unused individual product playbooks.

This is a cleanup change in prep for isolating master vs localhost tasks & vars.
This change removes unused playbooks that are not be part of the install or any test plans.
Removing these will avoid their use (accidental or otherwise) and
avoid confusion about their support as an installation method.

## Verification Steps
Install & uninstall should continue to work as normal.
This is only a refactor/cleanup change.
